packages/cursorless-engine/src/processTargets/modifiers/scopeHandlers/SentenceScopeHandler.ts

```diff
@@ -0,0 +1,49 @@
+import { Direction, Range } from "@cursorless/common";
+import { imap } from "itertools";
+import { NestedScopeHandler } from ".";
+import type { TargetScope } from "./scope.types";
+import { TokenTarget } from "../../targets";
+
+export default class SentenceScopeHandler extends NestedScopeHandler {
+  public readonly scopeType = { type: "sentence" } as const;
+  public readonly iterationScopeType = { type: "paragraph" } as const;
```

Comment:
👍

Reply:
By the way why do we need line eight? The constructor of nested scope handler makes that public already?
https://github.com/cursorless-dev/cursorless/blob/1046ddac884bd411770c78bc5b592a2da662ca2b/packages/cursorless-engine/src/processTargets/modifiers/scopeHandlers/NestedScopeHandler.ts#L55

Reply:
Got it. The risk here is just that we now define scope handlers in two different places that could in theory be different.

Reply:
sorry what do you mean "we now define scope handlers in two different places".  Which places are those?

Reply:
One scope gets passed into the constructor in `ScopeHandlerFactoryImpl.ts` and then another scope could be defined in the actual scope handler. I just thought this pattern was a bit peculiar.